_config/queuedjobs.yml

```diff
@@ -7,6 +7,7 @@ Injector:
   QueuedJobService:
     properties: 
       queueHandler: %$QueueHandler
+      queueRunner: %$DoormanRunner
```

Comment:
You can fix this by making runQueue and runJob take the service as a second parameter, then QueuedJob service can pass in `$this`.